tests/Microsoft.Identity.Test.Android.UIAutomation/androidTests.cs

```diff
@@ -62,8 +62,12 @@ public void RunAllTests()
                 AcquireTokenSilentTest,
 
                 AcquireTokenADFSV3InteractiveFederatedTest,
+                AcquireTokenADFSV3InteractiveManagedTest,
```

Comment:
There is no such thing as adfsv3 managed.

Reply:
I didn't think so but we had tests for them. This makes my day,! I will remove them!

Reply:
what were those testing? perhaps the name of the test was wrong?

Reply:
No, I think the lab API used to be confusing. You could query the lab to give you a user that was marked as "managed" and "Adfs" and the lab would return some user. We thought this is a valid permutation, but having discussed with the lab team, this is not. I believe the new API throws an exception if trying to get a "managed adfs" user.